modules/localStorage.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line
```

Comment:
- [ ] Please do not disable `eslint`, it will not show an error if you 

define class without exporting  => `class Books {} `
export a at the end  as default  => `export default Books;`